data/cargo-sources.json

```diff
@@ -0,0 +1,7743 @@
+[
```

Comment:
This is a really big file of hashes! Can this be auto-generated during flatpak build? Otherwise, it seems like this needs to be updated every time we bump our dependencies? In general adding a new build is also a request or a promise to maintain it. For something non-trivial, explanatory documentation about how to update support files is really useful.

Reply:
Exactly. Everything needs to be available offline and downloaded before the build begins (like the regular distros work). `data/flatpak_updater.sh` is here to regenerate it every time.

I would run `flatpak_updater.sh` on every push to keep the flatpak files synchronized.